CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## tip
 
+* BUGFIX: fix query display text in query history to show the actual expression instead of the full query object.
```

Comment:
```suggestion
* BUGFIX: fix query display text in query history to show the actual expression instead of the full query object. See [this issue](https://github.com/VictoriaMetrics/victorialogs-datasource/issues/194).
```